.github/workflows/build_branch33.yml

```diff
@@ -37,7 +37,8 @@ jobs:
       envs: >-
         {
           "SCALA_PROFILE": "scala2.13",
-          "PYTHON_TO_TEST": ""
+          "PYTHON_TO_TEST": "",
+          "ORACLE_DOCKER_IMAGE_NAME": "gvenzl/oracle-xe:21.3.0"
```

Comment:
The `build_and_test.yml` in branch-3.3 is configured as `oracle-xe:18.4.0`, but the previous daily test has always used `oracle-xe:21.3.0`, so it is still configured as `oracle-xe:21.3.0` here.